tool/tsh/tsh.go

```diff
@@ -366,6 +385,8 @@ func Run(args []string, opts ...cliOption) error {
 	login.Flag("overwrite", "Whether to overwrite the existing identity file.").BoolVar(&cf.IdentityOverwrite)
 	login.Flag("request-roles", "Request one or more extra roles").StringVar(&cf.DesiredRoles)
 	login.Flag("request-reason", "Reason for requesting additional roles").StringVar(&cf.RequestReason)
+	login.Flag("request-reviewers", "Suggested reviewers for role request").StringVar(&cf.SuggestedReviewers)
+	login.Flag("no-auto-request", "Disable auto-request behavior").BoolVar(&cf.NoAutoRequest)
```

Comment:
Can you explain why do we need `no-auto-request` flag?

Reply:
Is there a way we can avoid the extra flag?

Reply:
@klizhentas The `no-auto-request` flag is useful for debugging, but also potentially necessary as a malformed access request RBAC configuration or a misbehaving/offline plugin could otherwise lock people out of their cluster.  We could reduce clutter by making it a hidden flag, but I think its worth having.

@a-palchikov Tried your suggestion, but the result is confusing as help only shows the `auto-request` variant, which has no effect since the default is already true.  IMO this is unnecessarily confusing.

Reply:
@fspmarshall what does `no-auto-request` actually do?

Reply:
@klizhentas It disables the automatic request generation behavior triggered by `request_access: always`.  Normally, `tsh` checks the user's roles for a `request_access` directive and automatically generates a request if the directive is present.  This is a convenience behavior and not necessary for correct functioning of `tsh`.  Given that it has the potential to lock people out of their clusters if something goes wrong with their plugins or access request configuration, I think its a behavior that people should be able to disable.

Reply:
@fspmarshall what if we match 

`-request=auto|always|no`

With default value, it will figure out the reason, `--request=no` will behave similarly to agent loading behavior:

```
--add-keys-to-agent=auto|no
```

Reply:
Per discussion with @klizhentas elsewhere, we've decided to go with using `--request-roles=no` to tell `tsh` that it should avoid creating any requests *generally* during the login sequence.  If, for some reason, a user has a role name `no` and want to supply a single-element role list with that role, they can disambiguate via a trailing comma with `--request-roles=no,`.